app/models/bookshelf.rb

```diff
@@ -0,0 +1,15 @@
+require_dependency 'dpla/items'
+##
+# A model that represents a Book item retrieved from the search.
+#
+# For displaying the books in BookshelfController.
+# Search#conditions is overridden to forego the retrieval of facets.
+#
+# @see BookshelfController
+#
+class Bookshelf < Search
+
+  def conditions
+    { q: @term, facets: [], facet_size: 0 }.merge(@filters).merge(@args)
```

Comment:
@AudreyAltman you are correct about that.  I was confused because the `frontend` app has to have a cold restart in order for changes to take effect.  (Unicorn stop and start; can't give it a HUP signal.)  I thought that an earlier try at removing the parameters entirely wasn't working, as a result.